django/contrib/admin/filters.py

```diff
@@ -331,6 +331,15 @@ def __init__(self, field, request, params, model, model_admin, field_path):
                 self.lookup_kwarg_until: str(next_year),
             }),
         )
+        if field.null:
+            self.links += (
+                (_('No date'), {
+                    self.field_generic + 'isnull': str(1),
+                }),
+                (_('Has valid date'), {
+                    self.field_generic + 'isnull': str(0)
+                }),
```

Comment:
The only other `isnull`filter I could find uses `'True'`. Lets be consistent and use `'True'` and `'False`' instead of `'1'` and `'0'`.